test/presets/oracles/constrained/ConstrainedOracleToCompletionPercentageTest.sol

```diff
@@ -0,0 +1,174 @@
+    }
+
+    function testBetweenConstraintAtUpperBound() external {
+        MockConstrainedOracle oracleInstance = initializeConstrainedOracle(Constraint.Between, 10, 20);
+        assertEq(oracleInstance.toCompletionPercentage(20), 0);
+    }
+
+    function testBetweenConstraintAboveUpperBound() external {
+        MockConstrainedOracle oracleInstance = initializeConstrainedOracle(Constraint.Between, 10, 20);
+        assertEq(oracleInstance.toCompletionPercentage(40), 0);
+    }
+
+    function testNotBetweenConstraintBelowLowerBound() external {
+        MockConstrainedOracle oracleInstance = initializeConstrainedOracle(Constraint.NotBetween, 10, 20);
+        assertEq(oracleInstance.toCompletionPercentage(9), UNIT);
+    }
+
+    function testNotBetweenConstraintAtLowerBound() external {
+        MockConstrainedOracle oracleInstance = initializeConstrainedOracle(Constraint.NotBetween, 10, 20);
+        assertEq(oracleInstance.toCompletionPercentage(10), 0);
+    }
+
+    function testNotBetweenConstraintInRange() external {
+        MockConstrainedOracle oracleInstance = initializeConstrainedOracle(Constraint.NotBetween, 10, 20);
+        assertEq(oracleInstance.toCompletionPercentage(16), 0);
+    }
+
+    function testNotBetweenConstraintAtUpperBound() external {
+        MockConstrainedOracle oracleInstance = initializeConstrainedOracle(Constraint.NotBetween, 10, 20);
+        assertEq(oracleInstance.toCompletionPercentage(20), 0);
+    }
+
+    function testNotBetweenConstraintAboveUpperBound() external {
+        MockConstrainedOracle oracleInstance = initializeConstrainedOracle(Constraint.NotBetween, 10, 20);
+        assertEq(oracleInstance.toCompletionPercentage(40), UNIT);
+    }
+
+    function testRangeConstraintBelowLowerBound() external {
+        MockConstrainedOracle oracleInstance = initializeConstrainedOracle(Constraint.Range, 10, 20);
+        assertEq(oracleInstance.toCompletionPercentage(9), 0);
+    }
+
+    function testRangeConstraintAtLowerBound() external {
+        MockConstrainedOracle oracleInstance = initializeConstrainedOracle(Constraint.Range, 10, 20);
+        assertEq(oracleInstance.toCompletionPercentage(10), 0);
+    }
+
+    function testRangeConstraintInRange() external {
+        MockConstrainedOracle oracleInstance = initializeConstrainedOracle(Constraint.Range, 10, 20);
+        assertEq(oracleInstance.toCompletionPercentage(16), 600_000);
+    }
+
+    function testRangeConstraintAtUpperBound() external {
+        MockConstrainedOracle oracleInstance = initializeConstrainedOracle(Constraint.Range, 10, 20);
+        assertEq(oracleInstance.toCompletionPercentage(20), UNIT);
+    }
+
+    function testRangeConstraintAboveUpperBound() external {
+        MockConstrainedOracle oracleInstance = initializeConstrainedOracle(Constraint.Range, 10, 20);
+        assertEq(oracleInstance.toCompletionPercentage(40), UNIT);
+    }
+
+    function testLowerThanConstraintLessThanTarget() external {
```

Comment:
The test name here should be `testGreaterThanConstraintLessThanTarget`.

Reply:
Addressed in 085321c61ca8d07a7d23bc66030e6a08ff25b02e